runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java

```diff
@@ -30,19 +30,50 @@
 import net.runelite.client.config.ConfigItem;
 
 @ConfigGroup(
-	keyName = "tileindicators",
-	name = "Tile Indicators",
-	description = "Configuration for the tile indicators plugin"
+		keyName = "tileindicators",
```

Comment:
Fix your indentation settings (https://github.com/runelite/runelite/wiki/Code-Conventions). E.g continuation indent should be set to 4 not 8